src/Scripts/Ecs/Core/Entity.gd

```diff
@@ -4,6 +4,7 @@ const SpriteComponent = preload("res://Scripts/Ecs/Components/SpriteComponent.gd
 
 # type (as a string) => instance
 var _components = {}
+var sight_radius = 0
```

Comment:
Shouldn't this be in a component? Walls don't have sight, do they?

Reply:
It's true that walls don't have sight. Which component would sight go into? ... I guess `SightComponent` 👀